support-frontend/assets/pages/error/error404.tsx

```diff
@@ -4,7 +4,7 @@ import { renderPage } from 'helpers/rendering/render';
 import ErrorPage from './components/errorPage';
 
 // ----- Page Startup ----- //
-setUpTrackingAndConsents();
+setUpTrackingAndConsents({});
```

Comment:
It's okay to pass an empty object here because presumably we won't be running any A/B tests on this page?

Reply:
yes I've assumed AB testing is pointless on these types of pages!